content/id/docs/reference/glossary/image.md

```diff
@@ -4,15 +4,14 @@ id: image
 date: 2019-04-24
 full_link: 
 short_description: >
-  Instans yang disimpan dari sebuah kontainer yang mengandung seperangkat perangkat lunak yang dibutuhkan untuk menjalankan sebuah aplikasi.
+  Instans yang disimpan dari sebuah Container yang memuat seperangkat perangkat lunak yang dibutuhkan untuk menjalankan sebuah aplikasi.
```

Comment:
Same with PR #24427. We need to keep “instance“.

Reply:
@ariscahyadi do we use all capitalization for each term as Container, Pod, etc? Just to make sure